riak_test/oporset_test.erl

```diff
@@ -129,31 +117,3 @@ remove_test(Nodes) ->
     Result7=rpc:call(FirstNode, antidote, read,
                     [Key, Type]),
     ?assertMatch({ok, []}, Result7).
-    
-
-concurrency_test(Nodes) ->
```

Comment:
concurrency_test you mean?

That s a test that was actually testing nothing. I asked Li, who wrote it, and he told me it was created when there was no interdc module and he wrote it as a reminder, but it was actually never updated.

Manuel


> On 14 Sep 2015, at 13:43, Annette Bieniusa <notifications@github.com> wrote:
> 
> In riak_test/oporset_test.erl <https://github.com/SyncFree/antidote/pull/143#discussion_r39388596>:
> 
> > @@ -129,31 +117,3 @@ remove_test(Nodes) ->
> >      Result7=rpc:call(FirstNode, antidote, read,
> >                      [Key, Type]),
> >      ?assertMatch({ok, []}, Result7).
> > -    
> > -
> > -concurrency_test(Nodes) ->
> Why is this test no longer needed?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/143/files#r39388596>.
>